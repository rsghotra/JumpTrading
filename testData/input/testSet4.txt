3, 100000, 0, 2, 1075
0, 100001, 0, 3, 1080
0 100002 0 10 1075
0, abc, 0, 5, 1080
010000405970
0, 100005, 0, 10, 1080
1, 100004
a, b, c, d, e, f
0, 100007, b, 10, -1000
0, 100008, 0, -20, 1020
0, 100012, 0, 3000, 1100
0, 100012, 0, 3, 11200
0, , 0, 3000, 1100
0, jump, 0, 3000, 1100