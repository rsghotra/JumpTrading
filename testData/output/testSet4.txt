ERROR:  Bad Request: Invalid order type.
ERROR:  Bad Request: OrderId can not be zero/negative/out-of-range.
ERROR:  Bad Input: Unexpected or out of range datatype detected.
2,2,1080
3,100008
4,100001,1
ERROR:  Bad Input: Unexpected or out of range datatype detected.
ERROR:  Order with id 100004 does not exist.
ERROR:  Bad Input: Unexpected or out of range datatype detected.
ERROR:  Bad Input: Unexpected or out of range datatype detected.
ERROR:  Bad Request: Order size must be greater than zero.
ERROR:  Order with id 100012 already exists, not processing.
ERROR:  Bad Input: Unexpected or out of range datatype detected.
ERROR:  Bad Input: Unexpected or out of range datatype detected.