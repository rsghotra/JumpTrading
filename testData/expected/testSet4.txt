2,2,1080
3,100008
4,100001,1
